noir-projects/aztec-nr/aztec/src/note/utils.nr

```diff
@@ -20,8 +23,10 @@ fn compute_inner_note_hash<Note, N>(note: Note) -> Field where Note: NoteInterfa
     let header = note.get_header();
     let note_hash = note.compute_note_content_hash();
 
-    // TODO(#1205) Do we need a generator index here?
-    pedersen_hash([header.storage_slot, note_hash], 0)
+    pedersen_hash(
```

Comment:
Hello, we don't use poseidon everywhere because of [this](https://aztecprotocol.slack.com/archives/C04PUD9AA4W/p1713866556303739?thread_ts=1713798755.838109&cid=C04PUD9AA4W)